migrator/migration.py

```diff
@@ -135,18 +135,11 @@ def org_id(self):
             self._org_id = cf.get_org_id_for_space_id(self.space_id, self.client)
         return self._org_id
 
-    @property
```

Comment:
Why did this get deleted?

Reply:
in API v2, service_plan_visibility was its own object with an ID - so the relationship between service_plan x and organization y had a service_plan_visibility z. In order to make x stop being visible in y you had to look up z then call DELETE on it, like DELETE /v2/service_plan_visibilities/z

Now, service_plans have a visibilities property that is a list of objects (representing orgs and spaces). In order to make x no longer visible on y you DELETE the org on the service plan like DELETE /v3/service_plans/x/visibility/y

This property was about caching the value of z so it's no longer needed